src/plugins/mediaPlaybackSpeed/index.tsx

```diff
@@ -0,0 +1,97 @@
+/*
+ * Vencord, a Discord client mod
+ * Copyright (c) 2024 Vendicated and contributors
+ * SPDX-License-Identifier: GPL-3.0-or-later
+ */
+
+import "./styles.css";
+
+import { classNameFactory } from "@api/Styles";
+import { makeRange } from "@components/PluginSettings/components";
+import { Devs } from "@utils/constants";
+import definePlugin from "@utils/types";
+import { ContextMenuApi, FluxDispatcher, Menu, React, Tooltip } from "@webpack/common";
+import { RefObject } from "react";
+
+import SpeedIcon from "./components/SpeedIcon";
+
+const cl = classNameFactory("vc-media-playback-speed-");
+
+const speeds = makeRange(0.25, 3.5, 0.25);
+
+export default definePlugin({
+    name: "MediaPlaybackSpeed",
+    description: "Adds an icon to change the playback speed of media embeds",
+    authors: [Devs.D3SOX],
+
+    playbackSpeedComponent(mediaRef: RefObject<HTMLMediaElement>) {
+        const changeSpeed = (speed: number) => {
+            const media = mediaRef.current;
+            if (media) {
+                media.playbackRate = speed;
+            }
+        };
+
+        return (
+            <Tooltip text="Playback speed">
+                {({ onMouseEnter, onMouseLeave }) => (
+                    <button
+                        className={cl("icon")}
+                        onMouseEnter={onMouseEnter}
+                        onMouseLeave={onMouseLeave}
```

Comment:
this button has no accessible text, since it's just an svg without alt text. fairly certain the tooltip props contain an alt-text prop that's set to the tooltip text, so just spreading all props into your button should do the trick
```suggestion
                {tooltipProps => (
                    <button
                    	{...tooltipProps}
                        className={cl("icon")}
```